common/buildcraft/builders/container/ContainerQuarry.java

```diff
@@ -10,11 +10,9 @@
 import buildcraft.lib.gui.slot.SlotBase;
 
 public class ContainerQuarry extends ContainerBCTile<TileQuarry> {
-    private static final int PLAYER_INV_START = 50;
-
     public ContainerQuarry(EntityPlayer player, TileQuarry tile) {
         super(player, tile);
-        addFullPlayerInventory(PLAYER_INV_START);
+        addFullPlayerInventory(50);
```

Comment:
Why not inline player start in containers? are you sure that this is acutally needed? Because anyway there are many not extracted numbers in slot position declarations, so is there any point in player invenotry start constant? If for example texture changes, it will require to not only fix this contant, but all slot positions.